File: PYTHON/PetShop.py
#id, nama_produk, harga_produk, stok_produk, foto_produk

class PetShop:
    __id = ""
    __nama_produk = ""
    __harga_produk = 0
    __stok_produk = 0

    def __init__(self):
        self.__id = ""
        self.__nama_produk = ""
        self.__harga_produk = 0
        self.__stok_produk = 0

    def setId(self, id):
        self.__id = id
    def setNama(self, nama):
        self.__nama_produk = nama
    def setHarga(self, harga):
        self.__harga_produk = harga
    def setStok(self, stok):
        self.__stok_produk = stok

    def getId(self):
        return self.__id
    def getNama(self):
        return self.__nama_produk
    def getHarga(self):
        return self.__harga_produk
    def getStok(self):
        return self.__stok_produk

File: PHP/PetShop.php
<?php
class PetShop{
    private $id;
    private $nama_produk;
    private $harga_produk;
    private $stok_produk;
    private $foto_produk;

    public function __construct($id, $nama, $harga, $stok, $foto){
        $this->id = $id;
        $this->nama_produk = $nama;
        $this->harga_produk = $harga;
        $this->stok_produk = $stok;
        $this->foto_produk = $foto;
    }

    function setID($id){
        $this->id = $id;
    }
    function setNamaProduk($nama){
        $this->nama_produk = $nama;
    }
    function setHargaProduk($harga){
        $this->harga_produk = $harga;
    }
    function setStokProduk($stok){
        $this->stok_produk = $stok;
    }
    function setFotoProduk($foto){
        $this->foto_produk = $foto;
    }

    function getID(){
        return $this->id;
    }
    function getNamaProduk(){
        return $this->nama_produk;
    }
    function getHargaProduk(){
        return $this->harga_produk;
    }
    function getStokProduk(){
        return $this->stok_produk;
    }
    function getFotoProduk(){
        return $this->foto_produk;
    }
}
?>